packages/wrangler/src/publish.ts

```diff
@@ -48,15 +46,21 @@ export default async function publish(props: Props): Promise<void> {
     "A compatibility_date is required when publishing. Add one to your wrangler.toml file, or pass it in your terminal as --compatibility_date. See https://developers.cloudflare.com/workers/platform/compatibility-dates for more information."
   );
 
+  const triggers = props.triggers || envRootObj.triggers?.crons;
+  const routes =
+    props.routes ||
+    envRootObj.routes ||
+    (envRootObj.route ? [envRootObj.route] : undefined);
```

Comment:
Any reason why this was moved above the account_id check?

Reply:
the value of `routes` is needed for the default value of `deployToWorkersDev`